### PR TITLE
Add Default Values to Configure the Astro CLI

### LINKED ISSUE #1845: Add `Default values` to Astro CLI Config Doc
### Context

I was talking to @sunkickr through our Astro CLI docs and realized an opportunity to expand on our docs here: https://docs.astronomer.io/astro/cli/configure-cli?tab=astro

A couple of things:

- We don't specify the `default` values for these configs
- Some of the explanations for the configs should have examples that the user can read to help them decide if it's something they want to configure or not

### Ask

- Add a new column to `Available CLI Configurations` called `Default value` before `Possible values`
- For `show_warnings`, add `This includes a warning for when there's a new Astro Runtime version and a warning when there are no DAGs found in your Astro project`.
- Complete list of warnings (I don't _think_ we need to include all of these)
     - Warning for when there's a new Astro Runtime version
     - Warning when no DAGs are found in your Astro project
     - An invalid image name if you're not using Astro Runtime
     - A warning that tells you if you disable DAG-only deploys, your DAG

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1845

### CODE REVIEW

File: astro/cli/configure-cli.md [hunk 1]
```diff
@@ -53,43 +53,42 @@ The Astronomer product you're using determines the format and behavior of the co
...
-| `project.name`       | The name of your Astro project.         | Any string             |
-| `webserver.port`     | The port for the webserver in your local Airflow environment.          | Any available port             |
-| `show_warnings`      | Determines whether warning messages appear when starting a local Airflow environment.         | `true`, `false`             |
-| `upgrade_message`    | Determines whether a message indicating the availability of a new Astro CLI version displays in the Astro CLI.        | `true`, `false`             |
-| `page_size`             | Determines the size of the paginated response when `interactive` is set to `true`. The default is `20`.                      | Any integer             |
-| `verbosity`      | Determines the Astro CLI log level type. The default is `warning`.             | `debug`, `info`, `warning`, `error`             |
-| `webserver.port`     | The port for the webserver in your local Airflow environment.          | Any available port             |
+| Option              | Description | Default Value   | Possible Values   |
```
Comment (jwitz) [inline astro/cli/configure-cli.md:77]:
```suggestion
| Option              | Description | Default value   | Valid values   |
```

**Review by jwitz — APPROVED** [review]
Looks great!

**Comment by kwcanuck** [comment]
@paolaperaza did you want to complete a review before this is merged?

**Comment by jwitz** [comment]
Gonna merge this because the changes seem fairly straightforward.

**Pull request merged**